qiskit/aqua/operators/evolutions/evolved_op.py

```diff
@@ -88,9 +89,20 @@ def tensor(self, other: OperatorBase) -> OperatorBase:
 
         return TensoredOp([self, other])
 
-    def compose(self, other: OperatorBase) -> OperatorBase:
-        other = self._check_zero_for_composition_and_expand(other)
+    def _expand_dim(self, num_qubits: int) -> 'OperatorBase':
```

Comment:
I think the return type here can be `OperatorBase` directly right and does not need to be a string (a string is normally only needed for some forward ref. or ref to itself. There is another location just below and the same comment in a couple of other modules too where I do not think string is needed.